contributors/devel/release/issues.md

```diff
@@ -0,0 +1,193 @@
+# Targeting issues and PRs to release milestones
+
+This document describes how to target issues and PRs to a release.
+The process for shepherding issues into the release by the owner, release team, and GitHub bot
+is outlined below.
+
+## Definitions
+
+- *issue owners*: creator, assignees, and user who moved the issue it into a release milestone
+- *Y days*: refers to business days (using the location local to the release-manager M-F)
+- *post feature freeze* / *code slush*: starts when the master branch only accepts PRs for the release milestone
+
+## Requirements for adding an issue to the milestone
+
+**Note**: Issues with unmet label requirements will automatically be removed from the release milestone.
+
+When adding an issue to a milestone, the Kubernetes bot will check that the following
+labels are set, and comment on the issue with the appropriate instructions.  The
+bot will attempt to contact the issue creator 3 times (over 3 days)
+before automatically removing the issue from the milestone.
+
+Labels categories:
+
+- SIG label owner
+- Priority
+- Severity
+- Issue type
+
+### SIG owner label
+
+The SIG owner label defines the SIG to which the bot will escalate if the issue is not resolved
+or updated by the deadline.  If there are no updates after escalation, the
+issue may automatically removed from the milestone.
+
+e.g. `sig/node`, `sig/federation`, `sig/apps`, `sig/network`
+
+### Priority
+
+Priority label used by the bot to determine escalation path before moving an issues
+out of the release milestone.
+
+- `priority/critical-urgent`: Never automatically move out of a release milestone; continually escalate to contributor and SIG through all available channels.
+- `priority/important-soon`: Escalate to the issue owners and SIG owner; move out of milestone after several unsuccessful escalation attempts.
+- `priority/important-longterm`: Escalate to the issue owners; move out of the milestone after 1 attempt.
+
+### Severity
+
+The severity is used to triage which issues the release team needs to track as part of cutting
+a release.  As the release date approaches, release blocking issues are expected to be updated
+frequently and have an issue owner present at all burn down meetings until the issue is resolved.
+
```

Comment:
Yes they do - and the doc says they need to match.  I wrote this as having both labels because it makes it clear that only critical-urgent issues are release/blocker issues and all critical-urgent issues are release/blockers.  Do you think I should just drop the release/blocker labels?